src/components/countries/CountryList.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import Country from './Country'
+import styles from './CountryList.css'
+
+const CountryList = ({ countries, changePage, page }) => {
+    const countryElements = countries.map(country => (
+        <li key={country.code}>
+            <Link to={`/country/${country.code}`}>
+                <Country flagURL={country.flagURL} name={country.name} />
+            </Link>
+        </li>
+    ));
+
+    return (
+        <div>
+            <ul className={styles.controls}>
+                <li>
+                    <button
+                        onClick={() => changePage(-1)}>
+                        &lt;
+                    </button>
+                </li>
+                <li data-testid="pageNum">
+                    {page}
+                </li>
+                <li>
+                    <button
```

Comment:
you can add the disabled property to the button like 
```
disabled={page ===1}
```
that way the button is disabled if on the first page.

Reply:
That is a great idea, didn't even occur to me.  Thanks!